### DISCUSSION TITLE
Error "Debian Jammy not supported" on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by josmirpa** [telekom-security/tpotce#1229] [Q&A] [answered]
When running the installer using "Post-Install Auto Method" , i get an error message saying Aborting. Debian jammy is not supported.

Error occurs when trying to install apt-fast

I am currently running Ubuntu 22.04 . Thank you

**Comment by t3chn0m4g3** [comment, marked as answer]
Ubuntu is not supported.

**Reply by josmirpa** [reply]
Is it only compatible with debian? or is there any other way to install it on any other operating system?

Thanks for the answer!

**Reply by t3chn0m4g3** [reply]
[Debian 11](https://github.com/telekom-security/tpotce/blob/master/README.md#technical-concept) is T-Pot's base and nothing else is supported.
